kuksa_go_client/kuksa_client/grpc.go

```diff
@@ -83,7 +83,7 @@ func getArrayFromInput[T any](input string) ([]T, error) {
 	input = strings.TrimSuffix(strings.TrimPrefix(input, "["), "]")
 	
 	// Split the input string into separate values
-	pattern := `(?:\\.|[^",])*"(?:\\.|[^"])*"|'(?:\\.|[^']|\\')*'|[^",]+`
+	pattern := `(?:\\"|[^",])+|"(?:\\"|[^"])*"|'(?:\\"|[^']|\\')*'`
```

Comment:
In GOit is only possible to set a certain value through for example
```
"['dtc1, dtc2', dtc2, \"dtc3, dtc3\"]"
```
and this will result in 
```
{
    "path": "Vehicle.OBD.DTCList",
    "value": {
        "value": {
            "values": [
                "dtc1",
                "dtc2",
                "dtc2",
                "",
                "\"dtc3, dtc3\""
            ]
        },
        "timestamp": "2023-07-11T06:41:14.537216+00:00"
    }
}
```

Reply:
This seems odd, I will need to look into it a bit more. But then it is also so that it is a difference between how a string is set in code on what it actually will contain.

Reply:
With latest PR version the string /from main.go)

`var valstr = "['dtc1, dtc2', dtc2, \"dtc3, dtc3\", dtc4]"`

Results in:

```
Test Client> getValue Vehicle.OBD.DTCList
{
    "path": "Vehicle.OBD.DTCList",
    "value": {
        "value": {
            "values": [
                "dtc1, dtc2",
                "dtc2",
                "dtc3, dtc3",
                "dtc4"
            ]
        },
        "timestamp": "2023-07-12T06:11:15.227045+00:00"
    }
}

```